app/serializers/v1/keyword_serializer.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 
 module V1
-  class KeywordSerializer < ApplicationSerializer
-    attributes :content, :status, :created_at, :updated_at
+  class KeywordSerializer < V1::KeywordSimpleSerializer
+    attributes :result_page_html, :links_count
+
+    attributes :search_entries_info do |_, params|
```

Comment:
How about placing all these attributes at the root level like `links_count`?

```
attributes :top_ads_count do |_, params|
  params[:search_entries_query].top_ads_count
end

#...
```

It is a bit repetitive but it is a one time cost and it simplifies the response structure 💡

Reply:
Fixed in 94a8558a10fb388a0aede7696441c70800da97ea